example_config.yml

```diff
@@ -0,0 +1,13 @@
+objective:
+  - Lasso Regression[fit_intercept=False,reg=0.1]
+
+dataset: # dataset to run benchmark on
+  - leukemia
+
+solver: # list of example solvers to do benchmark
+  - celer
+  - cd
+  - sklearn
```

Comment:
is FISTA very slow in this context? Else, I would add it and remove `cd` to increase the diversity.

Reply:
It's true that cd and sklearn might be overlapping, but I think FISTA is quite slow for a quickstart purpose script, especially if we add leukemia data in?